### DISCUSSION TITLE
Seeing too many published releases in my feed

### DISCUSSION THREAD

**Discussion by PrayagS** [community/community#54467] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

So I don't know why but suddenly, I've started seeing a ton of published releases on the new feed.

All these releases are coming from repositories that are part of my org. But the reason stated is that `You're seeing this because you collaborated with <user in my org>`.

When I click on show less activity, I see the following options,
- I'm not interested in org/repo
- I'm not interested in published releases from <user in my org>
- I want to see fewer published releases

My ideal state is option 1 i.e. ignoring the repositories in my org. However, I've selected that option for almost all repositories in my org and have done so multiple times even for the same repository. But my feed still continues to show the releases.

How do I force it to ignore the repositories? I thought saying no once would be enough.

TIA.

**Comment by BasixKOR** [comment]
Perhaps this can help: #13148

**Reply by PrayagS** [reply]
I did go through that. That solution disables release notifications for all repos/users. I want to selectively disable it.

The whole `Show less activity like this` flow along with it asking for the reason made me believe that fine-grained ignore rules are possible. But as we can see, they're not working as expected.
